pkg/process/checks/process_test.go

```diff
@@ -446,3 +446,127 @@ func BenchmarkProcessCheck(b *testing.B) {
 		require.NoError(b, err)
 	}
 }
+
+func TestProcessCheckZombieToggleFalse(t *testing.T) {
+	processCheck, probe := processCheckWithMockProbe(t)
+	cfg := ddconfig.Mock(t)
+	processCheck.config = cfg
+	processCheck.skipZombieProcesses = processCheck.config.GetBool(configDisallowZombies)
+
+	now := time.Now().Unix()
+	proc1 := makeProcessWithCreateTime(1, "git clone google.com", now)
+	proc2 := makeProcessWithCreateTime(2, "mine-bitcoins -all -x", now+1)
+	proc3 := makeProcessWithCreateTime(3, "foo --version", now+2)
+	proc4 := makeProcessWithCreateTime(4, "foo -bar -bim", now+3)
+	proc5 := makeProcessWithCreateTime(5, "datadog-process-agent --cfgpath datadog.conf", now+2)
+	proc4.Stats.Status = "Z"
+	proc5.Stats.Status = "Z"
```

Comment:
Just a suggestion, but I think we can simplify this test and the other one a little bit by reducing the amount of processes you are creating. The amount of processes doesn't influence the behavior.